src/Evolution/Systems/GeneralizedHarmonic/Actions/NumericInitialData.hpp

```diff
@@ -236,8 +242,8 @@ struct SetNumericInitialData {
                     numeric_initial_data));
             *pi = std::move(
                 get<Tags::Pi<3, Frame::Inertial>>(numeric_initial_data));
-            *phi = std::move(
-                get<Tags::Phi<3, Frame::Inertial>>(numeric_initial_data));
+            // Set Phi to the numerical spatial derivative of spacetime_metric
+            partial_derivative(phi, *spacetime_metric, mesh, inv_jacobian);
```

Comment:
This overrides phi entirely so we shouldn't load it from the file. Is there a use case where we would want to load phi from file instead? Otherwise just delete it in line 72, and adjust related docs.